### PR TITLE
Fix/nex 81/polyfill issue with booklet rendering

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/NEX-81

Requires: 
 - [x] https://github.com/oat-sa/extension-tao-qtiprint/pull/58

Due to the changes in `tao-core` with the move to `ES2015`, the polyfill are not available anymore in dev mode. And that leads to issue in legacy browsers or tools that rely on legacy browsers, like it is for `wkhtmltopdf.

For that reason, the booklet generator must now be ran in production mode to be able to generate the PDF.

To make it working, in `config/generis.conf.php`, make sure `DEBUG_MODE` is set to `false`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ extension-tao-booklet
 
 An extension for TAO to create test booklets (publishable in MS-Word and PDF along with Answer Sheets)
 
+## Warning
+Due to the move of core libs to `npm` repositories, some code might not work on legacy browsers.
```
Comment (krampstudio) [inline README.md]: it's not linked to `npm` but to the update to ES2015

File: README.md [hunk 2]
```diff
@@ -3,6 +3,11 @@ extension-tao-booklet
...
+Especially for code that use to rely on polyfills, like for the `Promise`. The polyfills are now linked only when the code is bundled, and are not reachable anymore in development mode.
+For that reason, and because `wkhtmltopdf` is not supporting ES2015 and requires polyfills, the generation of PDF only works with bundled version (aka production mode).
```
Comment (krampstudio) [inline README.md]: (when the constant `DEBUG_MODE` is set to `false` in `config/generis.conf.php`)

File: README.md (hunk 1 above)
Comment (jsconan) [reply README.md]: yep my bad, will reword that

File: README.md (hunk 2 above)
Comment (jsconan) [reply README.md]: sure, I'll add more about it in the readme

**Pull request merged**